<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EloamCamera</class>
 <widget class="QWidget" name="EloamCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="3,1">
       <item>
        <widget class="QWidget" name="widget_image" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_log"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_deviceList">
             <property name="text">
              <string>设备列表</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_deviceList"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_videoMode">
             <property name="text">
              <string>视频模式</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_videoMode"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_resolution">
             <property name="text">
              <string>分辨率</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_resolution"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_openDev">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_closeDev">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_rotateLeft">
             <property name="text">
              <string>Rotate Left</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_rotateRight">
             <property name="text">
              <string>Rotate Right</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_leftRight">
             <property name="text">
              <string>Left-Right</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_upDown">
             <property name="text">
              <string>Up-Down</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QLabel" name="label_photoPath">
       <property name="text">
        <string>Photo Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_photoPath">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_browsePhoto">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_snapshot">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Snapshot</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
